File: ios/CalendarModule.m
//
//  CalendarManagerBridge.m
//  libtest
//
//  Created by matt on 1/17/21.
//

// RCTCalendarModule.m
#import "React/RCTBridgeModule.h"

@interface RCT_EXTERN_MODULE(CalendarModule, NSObject)

RCT_EXTERN_METHOD(createCalendarEvent:(NSString)name location:(NSString)location date:(NSString)date)

@end

File: ios/CalendarModule.swift
//
//  CalendarManager.swift
//  libtest
//
//  Created by matt on 1/17/21.
//

import Foundation

@objc(CalendarModule)
class CalendarModule: NSObject {

  @objc
  static func requiresMainQueueSetup() -> Bool {
    return true
  }
  
  @objc(createCalendarEvent:location:date:)
  func createCalendarEvent(_ name: String, location: String, date: String) -> Void {
//    let date = 999
    print("--------- Pretending to create an event on " + date + " at " + location )
  }
}
